### PR TITLE
Added `temp` persmissions for the deploy

### PR DESCRIPTION
**Pull request by aelfa**

### CODE REVIEW

**Comment by aelfa** [comment]
@doob187 Ignore if this was already added somewhere else.

File: traefik/installer/ubuntu.sh [hunk 1]
```diff
@@ -235,12 +235,17 @@ EOF
    for file in "$basefolder"/cloudflared/*.json
    do
    if [ ! -e $file ]; then
+   # Temp permission
+   sudo $(command -v chmod) 777 -R $basefolder/cloudflared
```
Comment (doob187) [inline traefik/installer/ubuntu.sh]: Remove sudo

File: traefik/installer/ubuntu.sh [hunk 2]
```diff
@@ -235,12 +235,17 @@ EOF
...
    $(command -v docker) pull cloudflare/cloudflared:2021.8.7-amd64
    $(command -v docker) run -it --rm -v $basefolder/cloudflared:/home/nonroot/.cloudflared/ cloudflare/cloudflared:2021.8.7-amd64 tunnel login
    # create
    $(command -v docker) run -it --rm -v $basefolder/cloudflared:/home/nonroot/.cloudflared/ cloudflare/cloudflared:2021.8.7-amd64 tunnel create dockserver
    fi
    done
+   # permission fix
+   sudo $(command -v chown) -hR 1000:1000 $basefolder/cloudflared
```
Comment (doob187) [inline traefik/installer/ubuntu.sh]: Remove sudo 

Not needed

**Review by doob187 — CHANGES_REQUESTED** [review]
See notes

**Review by doob187 — APPROVED** [review]

**Pull request merged**